iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -72,6 +72,16 @@ <h3 data-i18n="inspector.filter">Filter</h3>
               <option value="text" data-i18n="inspector.textOption">Text</option>
           </select>
 
+          <div class="control-label" data-i18n="inspector.calculations">
+            Calculations
+          </div>
+          <select id="search-filter-calculations">
+            <option value="*" data-i18n="inspector.allOption">ALL</option>
```

Comment:
The distinction between "ALL" and "Either" seems somewhat confusing.  I wonder if empty string or "(No filter)" would be clearer than "ALL" (here and above), and I wonder if explicitly "Summation or Contributor" would be clearer than "Either".

Calculations are defined in terms of the summation, so it might be more natural to list "Summation" first. (In calculation 1.1, the summation is called total, but it seems fine to keep it as-is for now.)

I'm not sure "Item" part adds much, so you might consider dropping it.

Reply:
Agreed and applied.

For "ALL", I felt that just `-` was better than an empty string, it gives something more clear to click on in the dropdown (at least the way it's rendered on Mac), but this certainly isn't my expertise so I'll concede if anyone has strong opinions.